framework/freedomotic-core/src/test/java/com/freedomotic/rules/StatementTest.java

```diff
@@ -0,0 +1,108 @@
+package com.freedomotic.rules;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StatementTest {
+
+  @Test
+  public void createLogicalInputValid() {
+    final Statement statement = new Statement();
```

Comment:
remove the otherwise unused `statement` declaration, inline into line 11 as `retval = new Statement().create(...`